WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingTaskView.swift

```diff
@@ -114,8 +126,15 @@ struct StoreOnboardingTaskView_Previews: PreviewProvider {
     static var previews: some View {
         VStack {
             Group {
-                StoreOnboardingTaskView(viewModel: .init(task: .customizeDomains, isComplete: false, icon: .domainsImage), onTap: { _ in })
-                StoreOnboardingTaskView(viewModel: .init(task: .customizeDomains, isComplete: true, icon: .domainsImage), onTap: { _ in })
```

Comment:
super nit: is it worth keeping one task with `isComplete: true`?

Reply:
Done in https://github.com/woocommerce/woocommerce-ios/pull/8948/commits/53a3b14e645dfcb2e16c49a8a8555f9fee4c536e